### PR TITLE
pie-stylelint-config@v0.1.0 - rules and project setup

### PR DESCRIPTION
**Pull request by jamieomaguire**
v0.1.0
------------------------------
*January 11, 2023*

### Added
- Base rules
- Project skeleton

### CODE REVIEW

File: packages/tools/pie-stylelint-config/LICENCE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Apache License
+Version 2.0, January 2004
+http://www.apache.org/licenses/
+
+Copyright (c) Just Eat Holding Ltd
```
Comment (ashleynolan) [inline packages/tools/pie-stylelint-config/LICENCE.md:5]: I was looking at this yesterday – I think all our licenses should be `Copyright (c) Just Eat Takeaway` now.

We should change any going forward that we notice (but especially new ones we add)

File: packages/tools/pie-stylelint-config/README.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+```bash
+npm install @justeattakeaway/pie-stylelint-config --save-dev
+```
+
+## Usage
+
+### Rules
+Currently there are 3 different sets of rules:
+1. `base` - the base rules that we'd advise all projects use and extend as needed.
```
Comment (ashleynolan) [inline packages/tools/pie-stylelint-config/README.md:26]: Super NIT's:

```suggestion
### Rules

Currently, there are 3 different sets of rules:

1. `base` - the base rules that we'd advise all projects use and extend as needed.
```

File: packages/tools/pie-stylelint-config/README.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+{
+  "extends": "/absolute/path/to/@justeattakeaway/pie-stylelint-config/base"
+}
+```
+
+### Extending the config
+
+Simply add a `"rules"` key to your config, then add your overrides and additions there.
+
+For example, to change the `indentation` to tabs, turn off the `number-leading-zero` rule:
```
Comment (ashleynolan) [inline packages/tools/pie-stylelint-config/README.md:74]:
```suggestion
For example, to change the `indentation` to tabs, and turn off the `number-leading-zero` rule:
```

**Review by ashleynolan — COMMENTED** [review]
Couple of minor things – otherwise LGTM

File: packages/tools/pie-stylelint-config/LICENCE.md (hunk 1 above)
Comment (jamieomaguire) [reply packages/tools/pie-stylelint-config/LICENCE.md]: updated - let me know if it still needs the `Holding Ltd` part - wasn't sure from your comment!

**Review by ashleynolan — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**